Meadowlark-js/packages/meadowlark-core/src/security/JwtValidator.ts

```diff
@@ -41,6 +41,17 @@ export function validateJwt(authorizationHeader: string | undefined): JwtValidat
     };
   }
 
+  if (jwtStatus.authorizationStrategy === '') {
```

Comment:
This overall works, but there is an issue (maybe?) - when you have an invalid token (for example the token doesn't parse at all) the error returned is this one, even if this is not the sole reason for the bad token.

Reply:
I'm going to drop a link to this comment in the tech docs page where I'll be working on design plans for full oauth2 support. Agreed this is not great, and that now is not the time to improve it.

Reply:
AuthorizationStrategy has an UNDEFINED (not sure how I missed that)  so the highlighted line looks like `jwtStatus.authorizationStrategy === 'UNDEFINED'`